### PR TITLE
Test Incentive E2E

### PR DESCRIPTION
**Pull request by opendansor**
E2E test for incentive values with manually setting the network weights.

### CODE REVIEW

File: tests/e2e_tests/multistep/test_incentive.py [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+    assert bob_neuron.trust == 1
+
+    alice_neuron = metagraph.neurons[0]
+    assert alice_neuron.validator_permit is True
+    assert alice_neuron.dividends == 1
+    assert alice_neuron.stake.tao == 9999.999999
+    assert alice_neuron.validator_trust == 1
+
+
+def wait_epoch(interval, subtensor):
```
Comment (basfroman) [inline tests/e2e_tests/multistep/test_incentive.py:294]: nitpick: Usually functions-helpers are placed before the functions that call it. It's just easier to navigate.

File: tests/e2e_tests/multistep/test_incentive.py [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+    setup_wallet,
+    template_path,
+    repo_name,
+)
+
+logging.basicConfig(level=logging.INFO)
+
+
+@pytest.mark.asyncio
+async def test_incentive(local_chain):
```
Comment (basfroman) [inline tests/e2e_tests/multistep/test_incentive.py:45]: nitpick: I know you put comments across the test, but adding single-line docstring could looks elegant and helpful

**Review by basfroman — APPROVED** [review]

**Review by thealligatorking — APPROVED** [review]

**Review by ibraheem-abe — APPROVED** [review]
Nice!

**Comment by distributedstatemachine** [comment]
Team opendansooorrrrr 🚀🚀🚀🚀🚀🚀

File: tests/e2e_tests/multistep/test_incentive.py (hunk 2 above)
Comment (opendansor) [reply tests/e2e_tests/multistep/test_incentive.py:45]: Good point! I added comments to all E2E tests I have added so far!

**Pull request merged**